sound/soc/sof/debug.c

```diff
@@ -115,7 +116,37 @@ int snd_sof_debugfs_create_item(struct snd_sof_dev *sdev,
 
 	return 0;
 }
-EXPORT_SYMBOL(snd_sof_debugfs_create_item);
+EXPORT_SYMBOL(snd_sof_debugfs_io_create_item);
+
+/* create FS entry for debug files to expose kernel memory */
+int snd_sof_debugfs_buf_create_item(struct snd_sof_dev *sdev,
+				    void *base, size_t size,
+				    const char *name)
+{
+	struct snd_sof_dfsentry_buf *dfse;
+
+	if (!sdev)
+		return -EINVAL;
+
+	dfse = kzalloc(sizeof(*dfse), GFP_KERNEL);
```

Comment:
Can you use the latest code, this was moved to:
dfse = devm_kzalloc(sdev->dev, sizeof(*dfse), GFP_KERNEL);

Reply:
@plbossart done